libs/openFrameworks/graphics/ofImage.cpp

```diff
@@ -182,16 +182,21 @@ static int getJpegOptionFromImageLoadSetting(const ofImageLoadSettings &settings
 template<typename PixelType>
 static bool loadImage(ofPixels_<PixelType> & pix, const of::filesystem::path& _fileName, const ofImageLoadSettings& settings){
 	ofInitFreeImage();
-
+	
+#if defined(TARGET_WIN32) || defined(__MINGW64__) || defined(__CYGWIN__) || defined(TARGET_WINVS)
+	#define OF_TARGET_WINDOWS
```

Comment:
is this the place to `#define` the platform? it seems what happens in `ofConstants.h` should be used and not duplicated here?

Reply:
Yes sure. this was the motivation of opening that discussion about WIN32 and a more organized define. 
something that can represent when OS Windows are being used

Reply:
ah! well yes then I like your `OF_${CATEGORY}_${TAG}` pattern at the end of #7450.

maybe for the sake of the FS/wstring work, simply add a `#define OF_OS_WINDOWS` at line 89 of `ofConstants.h` and work with that for now? (or use `TARGET_WIN32` knowing a refactor is coming)

Reply:
https://github.com/openframeworks/openFrameworks/pull/7495